### ISSUE TITLE
Access to parent field through __ does not work

### ISSUE THREAD

**Issue by robertkowalski1974** [viewflow/django-material#498]
In One to Many relations I was accessing Parent fields with __.
For example if Child had Parent then I can have:

`list_display =['Parent__ParentField']`

This does not seem to be working with django-material ModelViewSet

**Comment by kmmbvnr** [comment]
Yep, looks like it's requires to fix lookup procedure. I hope i would add this ti 2.0 only.

Currently this can be overcome by using custom method

```
    def parentfield(self, obj):
        return obj.parent.parentfield
```
